clang/test/CIR/CodeGen/vtable-rtti.cpp

```diff
@@ -17,12 +17,6 @@ class B : public A
     virtual ~B() noexcept {}
 };
 
-// Type info B.
```

Comment:
The type aliasing is pretty useful for readability, (a) why are they gone? (b) should we have aliases that reflect the 3 struct styles?

Reply:
a. The way these aliases were generated seemed more of a bug than a feature.
b. It might be an interesting idea, however, I believe two kinds of aliases would suffice:

  - **Named records**: simply use the unique name as an alias.
  - **Anonymous records:** return the alias as `ty_anon` and let MLIR solve conflicts.
  
The first is already implemented. The second seems simple enough to add in this PR. I'll patch it.

Reply:
@bcardosolopes applied.